py/desiperf/instperfapp/pages/focalplane.py

```diff
@@ -48,22 +49,25 @@ def get_data(self, attr1, attr2, update=False):
             self.plot_source = ColumnDataSource(data_)
 
     def page_layout(self):
-        this_layout = layout([[self.page.header],
+        this_layout = layout([[self.plots.header],
                               [self.x_select, self.y_select, self.btn],
                               [self.corr, self.details, self.cov],
                               [self.ts1],
                               [self.ts2]])
-        tab = Panel(child=this_layout, title=self.page.title)
+        tab = Panel(child=this_layout, title=self.plots.title)
         return tab
 
     def time_series_plot(self):
-        self.corr = figure(plot_width=350, plot_height=250, tools=self.page.tools, x_axis_label=self.x_select.value, y_axis_label=self.y_select.value)
-        self.ts1 = figure(plot_width=900, plot_height=200, tools=self.page.tools, active_drag="xbox_select", x_axis_label='MJD_OBS', y_axis_label=self.x_select.value)
-        self.ts2 = figure(plot_width=900, plot_height=200, tools=self.page.tools, active_drag="xbox_select", x_axis_label='MJD_OBS', y_axis_label=self.y_select.value)
+        self.corr = self.plots.figure(width=350, height=250, x_axis_label='attr1', y_axis_label='attr2')
```

Comment:
I'm not sure why you switched to the labels 'attr1' and 'attr2' - I don't find those that informative!

Reply:
I agree. I think we will want to change this in the future. My problem at this point is that I can't get the x_axis_label to change when they are re-selected.

Reply:
I ran into that too. Meant to flag it as a problem.